src/mesh/unstructured_mesh.C

```diff
@@ -56,7 +56,6 @@ UnstructuredMesh::UnstructuredMesh (const Parallel::Communicator & comm_in,
 }
 
 
-
```

Comment:
This blank line deletion shouldn't be part of the diff.

Reply:
Sorry, that one's my fault.  I caught and fixed one of these in boundary_info.h after I finished squashing commits but I didn't think to check elsewhere afterwards.